cmake-modules/AzureDoxygen.cmake

```diff
@@ -17,19 +17,33 @@ function(generate_documentation PROJECT_NAME PROJECT_VERSION)
         set(DOXYGEN_OUTPUT_DIRECTORY docs)
         set(DOXYGEN_LAYOUT_FILE ${CMAKE_SOURCE_DIR}/eng/docs/api/assets/DoxygenLayout.xml)
         set(DOXYGEN_RECURSIVE YES)
+        if (MSVC)
+          set(DOXYGEN_WARN_FORMAT "$file($line) : $text")
+        endif()
+#        set(DOXYGEN_WARN_AS_ERROR FAIL_ON_WARNINGS)
```

Comment:
uncomment?

Reply:
I have an outstanding request to the ES team to upgrade to a current version of doxygen. Until then, we need to disable the failures :(.